tensorflow/compiler/xla/service/gpu/gpu_fusible.cc

```diff
@@ -172,7 +172,8 @@ bool IsLoopFusible(const HloInstruction& instr) {
           instr.opcode() == HloOpcode::kDynamicSlice ||
           instr.opcode() == HloOpcode::kDynamicUpdateSlice ||
           (instr.opcode() == HloOpcode::kFusion &&
-           instr.fusion_kind() == HloInstruction::FusionKind::kLoop) ||
+           instr.fusion_kind() == HloInstruction::FusionKind::kLoop &&
+	   !instr.IsMultiOutputFusion()) ||
```

Comment:
Is see. This is because of the GTE insertion, correct? So we can fuse into a multioutput node but cannot fuse a multioutput node into a different one.